src/main/java/no/nav/foreldrepenger/los/oppgave/tilbudtoppgave/TilbudtOppgaveObserver.java

```diff
@@ -0,0 +1,27 @@
+package no.nav.foreldrepenger.los.oppgave.tilbudtoppgave;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.event.Observes;
+import jakarta.enterprise.event.TransactionPhase;
+import jakarta.inject.Inject;
+import jakarta.persistence.EntityManager;
+
+@ApplicationScoped
+public class TilbudtOppgaveObserver {
+    private EntityManager entityManager;
+
+    public TilbudtOppgaveObserver() {
+    }
+
+    @Inject
+    public TilbudtOppgaveObserver(EntityManager entityManager) {
+        this.entityManager = entityManager;
+    }
+
+    // fpsak med venner bruker som regel default TransactionPhase.IN_PROGRESS, vurder hva som blir korrekt her
+    public void handle(@Observes(during = TransactionPhase.AFTER_SUCCESS) TilbudtOppgave event) {
```

Comment:
Jeg er lunken til CDI-events (og Hibernate-events) pga lesbarhet + navigerbarhet. Ganske lunken til updates gjort i Observers. Men å gjøre update etter Tx-success (commit?) er noe annet - hva er Tx-kontext for update her, hva skjer med lås-venting, rollback, feilhåndtering etc? 
Vil egentlig ikke vite svarene og synes ikke vi skal gå denne veien - skal noe oppdateres, gjør det helst i OpgvDtoTjeneste. Tx-kontext er primært controller eller tasks, evt lagre ny task for delayed, asynch update. Hele teamet.